corehq/apps/reports/standard/monitoring.py

```diff
@@ -337,6 +337,31 @@ def sort_column(self):
         else:
             return "landmark_%d" % (landmark,)
 
+    @property
+    def sort_key(self):
```

Comment:
actually don't need almost any of this https://github.com/dimagi/commcare-hq/pull/12500/commits/ec3dfa3772332ff8d4e6d4270324a14696d5c246